run-integration-tests.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+set +eux pipefail
```

Comment:
Why do you need pipefail here? Not seeing any pipelines.

Reply:
I guess we don't. Was a copilot auto complete

Reply:
I suspected as much, but didn't want to accuse you of committing AI code you didn't understand.

To be fair I asked the AI to explain what this line did in order to leave that comment.